resources/views/movies/create.blade.php

```diff
@@ -1,58 +1,93 @@
 
-                    <!--Poster-->
-                    <div>
-                        <x-label for="poster" value="Poster"/>
+                                <x-input class="border-pink-500 border bg-purple-800 bg-opacity-25"
+                                type="text"
+                                name="poster"
+                                id="poster"
+                                :value="old('poster')"/>
+                            </div>
+                            <!--Trailer-->
+                            <div>
+                                <x-label for="trailer" value="Trailer"/>
 
-                        <x-input type="text" name="poster" id="poster" :value="old('poster')"/>
-                    </div>
+                                <x-input class="border-pink-500 border bg-purple-800 bg-opacity-25"
+                                type="text"
+                                name="trailer"
+                                id="trailer"
+                                :value="old('trailer')"/>
+                            </div>
+                        </div>
 
-                    <!--Trailer-->
-                    <div>
-                        <x-label for="trailer" value="Trailer"/>
+                        <!--cell 2-->
+                        <div class="justify-self-center grid grid-cols-1">
+                            <!--Blurb-->
+                            <div class="self-end">
+                                <x-label for="blurb" value="Blurb"/>
 
-                        <x-input type="text" name="trailer" id="trailer" :value="old('trailer')"/>
-                    </div>
+                                <textarea class="leading-normal border-pink-500 border bg-purple-800 bg-opacity-25"
+                                name="blurb"
+                                id="blurb"></textarea>
+                            </div>
+                        </div>
 
-                    <!--Blurb-->
-                    <div>
-                        <x-label for="blurb" value="Blurb"/>
+                        <!--cell 3-->
+                        <div class="pb-8 justify-self-center">
+                            <!--Celebrities-->
+                            <x-label for="celebrity" value="Celebrities"/>
 
-                        <textarea name="blurb" id="blurb"></textarea>
+                            <textarea class="leading-normal border-pink-500 border bg-purple-800 bg-opacity-25"
+                            name="celebrities"
+                            id="celebrities"></textarea>
+                        </div>
 
-                    </div>
+                        <!--cell 4-->
+                        <div class="pb-8 justify-self-center">
+                            <!--Genre-->
+                            <x-label for="genre" value="Genres"/>
 
-                </div>
+                            <textarea class="leading-normal border-pink-500 border bg-purple-800 bg-opacity-25"
```

Comment:
Same as the comment above ;D. 

How I would do it, I would get all the genres in the controller and then have a for each loop here to show the genres list in the form. Inside the loop, I would associate the value of it with the id of the genre. In the request, I would pass an array of genres_ids and I would validate them. If the validation passes, I would use the `syncWithoutDetaching()` method from the [Laravel docs](https://laravel.com/docs/8.x/eloquent-relationships#syncing-associations) to link the movie with the genres id I selected.